module_3/src/schema/validation.js

```diff
@@ -4,15 +4,15 @@ import statusCodes from '../../statusCodes.js';
 export function userValidation(req, res, next) {
```

Comment:
I'd extract some low level common function or middleware which just accepts all necessary data and invokes `validate` method on given schema provided you follow the same way of payload validation

Reply:
im not sure that i understand correctly what your mean, is that something like this?

![image](https://user-images.githubusercontent.com/51543491/125797715-a80401af-2005-4462-82e3-3ab7b5487f86.png)
![image](https://user-images.githubusercontent.com/51543491/125797762-bd7cb356-e94c-43b9-a469-c07bc6642b00.png)

Reply:
something like that